config/test-realsense.json

```diff
@@ -2,19 +2,14 @@
   "version": 2,
   "robot": {
     "modules": {
-      "rs": {
+      "app": {
           "driver": "realsense",
-          "in": ["trigger"],
-          "out": ["pose2d"],
-          "init": {}
-      },
-      "timer": {
-          "driver": "timer",
-          "in": [],
-          "out": ["tick"],
-          "init": {"sleep": 0.1}
+          "init": {
+            "depth_subsample": 3,
+            "pose_subsample": 20
```

Comment:
20 means 250/20 = 12.5Hz? More user friendly would be frequency specification but ... next iterration

Reply:
No, the pose is 200Hz - I am not sure about different UI - I don't want to advertise the ability to set arbitrary frequency when that is not possible. Here we say "take each 20th sample" which is something different from arbitrary frequency.